lib/logstash/inputs/kafka.rb

```diff
@@ -123,8 +123,11 @@ class LogStash::Inputs::Kafka < LogStash::Inputs::Base
   config :session_timeout_ms, :validate => :string
   # Java Class used to deserialize the record's value
   config :value_deserializer_class, :validate => :string, :default => "org.apache.kafka.common.serialization.StringDeserializer"
-  # A list of topics to subscribe to.
-  config :topics, :validate => :array, :required => true
+  # A list of topics to subscribe to, defaults to ["logstash"].
+  config :topics, :validate => :array, :default => ["logstash"]
```

Comment:
The reason for default and not required was because, it is kind of either or topics_pattern or topics needs to be used. If topics_pattern is used the topics configuration is ignored and doesn't make sense to put in a required dummy value for that. Similar to Elasticsearch where there are defaults for index an none are required.